javascript/src/seedsource/containers/Map.jsx

```diff
@@ -610,13 +579,42 @@ class Map extends React.Component {
         }
     }
 
-    updateVectorLayers(layers) {
-        let numLayersToAdd = layers.length - this.displayedVectorLayers.length
+    updateRasterLayers(layers) {
+        let numLayersToAdd = layers.length - this.displayedRasterLayers.length
+        let { objective, climate, region } = this.props
+        let { serviceId } = this.props.job
+
+        if (numLayersToAdd > 0) {
+            this.displayedRasterLayers.push(...Array(numLayersToAdd).fill().map((_, index) => {
+                let layer = layers[this.displayedRasterLayers.length+index]
+                let url = getLayerUrl(layer, serviceId, objective, climate, region)
+                return L.tileLayer(`/tiles/${url}/{z}/{x}/{y}.png`, {zIndex: 1, opacity: 1}).addTo(this.map)
+            }))
+        } else if (numLayersToAdd < 0) {
+            this.displayedRasterLayers
+                .splice(numLayersToAdd, Math.abs(numLayersToAdd))
+                .forEach(layer => this.map.removeLayer(layer))
+        }
 
+        if (layers.length) {
+            let url
+            layers.forEach((layer, index) => {
+                url = `/tiles/${getLayerUrl(layer, serviceId, objective, climate, region)}/{z}/{x}/{y}.png`
```

Comment:
Since this is scoped, you don't need to define `let url` outside. Just define it inline here:

```javascript
let url = `/tiles/...`
```